lang/en/texts/android.html

```diff
@@ -0,0 +1,30 @@
+<h4>Mobile</h4>
+<a download="4. Additives - Mobile - EN.png" href="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png" target="_blank"><img width="300px" src="https://world.openfoodfacts.org/files/presskit/PressKit/android/screenshots/4. Additives - Mobile - EN.png"></a>
```

Comment:
Due to the full-size images (which I understand are supposed to be used for some kind of press information package?) being used as the `img.src`, loading this page uses a whopping 22.2 MB of traffic. That might be a very unpleasant surprise and experience for people opening it on a phone or otherwise metered connection. Would it be possible to use smaller (both in size, and compression) thumbnails for preview purposes?